pages/en/node-operators/transaction-failures.mdx

```diff
@@ -0,0 +1,53 @@
+import Page from "@reason/pages/Docs";
+import DocLink from "@reason/components/DocLink";
+export default Page ({ title: "Transaction failures" });
+
+# Transaction failures
+
+If a transaction fails to be applied, usually it would trigger 1 or more failures. There's 3 kind of transactions: payments, delegations, or zkApp transactions. There're several different kinds of failure in mina system. Some of the failures are specific to some kind of transactions. Let me explain them one by one below.
+
+## Source_not_present
+For payment, this means the mina node can not find the sender's account in the ledger.
+For delegation, this means the mina node can not find the delegator's account in the ledger.
+
+## Receiver_not_present
+This kind of failure only happens when a user tries to delegates to an non-exist account. If a user tries to send a payment to a new account, then this account would be created by this payment transaction.
+
+## Amount_insufficient_to_create_account
+For payment, this means the user tries to create a new account in this payment transaction, but the amount in the transaction is not enough for covering the account creation fee.
+For zkApp transaction, this means the smart contract tries to create 1 or more new accounts, but the amount in the smart contract is not enough for covering the account creation fee(s).
+
+## Source_insufficient_balance
+For payment, this means the sender does not have enough balance to cover the amount in the transaction.
+
+## Source_minimum_balance_violation
+For payment, this means the sender is a time-locked accounts and the unlocked funds is not enough to cover the amount in the transaction.
+
+For zkApp trasaction, this means 1 or more accounts in the zkApps are time-locked accounts and they violated the minimum balance requirement (trying to use more funds than unlocked at the moment).
+
+## Token_owner_not_caller
+This failure is unique to zkApps. You can think of zkApp transactions as a smart contract that update several different accounts, then it's possible to have updates to accounts with non-default tokens (tokens that owned by some specific accounts). Those are usually called token transfers/transactions by engineers. The token transactions must be directly called by token owners; or any immediate calls must 'delegate' the call's ownership to the token owner. This failure would be triggered if the above conditions are not satisfied.
+
+## Overflow
+The resulting balance of the transaction is too large to store.
+
+## Global_excess_overflow / Local_excess_overflow
+These 2 failures are unique to zkApps. They mean overflow happens when computing the balance changes for accounts that are involved in the zkApp.
```

Comment:
two

(the rule I follow: small numbers should be spelled out, large numbers can be presented as digits)